index.html

```diff
@@ -0,0 +1,2199 @@
+
+    <nav class=" nav-bar navbar-expand-lg navbar-light navigation border-bottom">
+      <div class=" collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="d-flex flex-row align-items-center nav-list font-weight-bold text-body justify-content-between">
+          <li class="nav-item"><a class="border-right" href="#">U.S</a></li>
+          <li  class="nav-item"><a class="border-right" href="#">World</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Business</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Tech & Science</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Culture</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Newsgeek</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Sports</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Health</a></li>
+          <li class="nav-item"><a class="border-right" href="#">The Debate</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Vantage</a></li>
+          <li class="nav-item"><a  href="#">Weather</a></li>
+          <li  class="nav-item nav-icons noPadding text-body d-flex flex-row">
+            <input class=" form-control border-0 search_input w-100" type="search" placeholder="Search" aria-label="Search">
+              <i class="fa fa-search text-body search-icon " aria-hidden="true"></i>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <main class="container-fluid main-wrapper">
+      <!-- first section small screen -->
+      <section class="top-story  d-sm-block d-md-none d-lg-none">
+        <a  class="first-section-title w-100" href="#">
+          <h2 class="font-weight-bolder">TOP STORY</h2>
+        </a>
+        <article class="border-0 top-story-article">
+          <div>
+            <a href="#">
+              <img class="rounded-0 w-100" src="https://d.newsweek.com/en/full/1581101/china-aircraft-carrier-liaoning-pacific.jpg" alt="President">
+            </a>
+            <a href="#">
+              <p class="politics">POLITICS</p>
+            </a>
+          </div>
+          <div class="article-body">
+            <a href="#">
+              <h2 class="mag_link2 article-title pt-2">
+                Chinese Carrier Sails into Pacific, Media Mock U.S. Navy's COVID-19 Woes
+              </h2>
+            </a>
+            <p class="top-story-paragraph">
+              The White House coronavirus response coordinator displayed a graph
+               that shows the mortality rates of countries around the world, 
+               and China is at the bottom with the lowest rate.
+            </p>
+          </div>
+        </article>
+      </section>
+
+      <section class="featured-stories d-sm-block d-md-none d-lg-none border-bottom">
+        <a  class="first-section-title w-100" href="#">
+          <h2 class="font-weight-bolder">FEATURED STORIES</h2>
+        </a>
```

Comment:
On the mobile version you are repeating this section before More stories. That shouldn't happen.